docs/advanced-entity-analytics/turn-on-risk-engine.asciidoc

```diff
@@ -0,0 +1,80 @@
+[[turn-on-risk-engine]]
+== Turn on the risk scoring engine
+
+beta[]
+
+IMPORTANT: To use Entity Risk Scoring, your role must have the appropriate privileges. For more information, refer to 
+
+// <<ers-requirements, Entity Risk Scoring prerequisites>>.
+
+The latest risk scoring engine, available in {stack} version 8.11 and newer, runs hourly to aggregate `Open` and `Acknowledged` <<alerts-ui-manage, alerts>> from the last 30 days, and assigns risk score to the host or user. It then aggregates the individual risk scores and normalizes them to a 0-100 range. The engine assigns a risk level by mapping the normalized risk score to one of these levels:
+
+[width="100%",options="header"]
+|==============================================
+|Risk level |Risk score
+
+|Unknown |< 20
+|Low |20-40
+|Moderate |40-70
+|High     | 70-90
+|Critical  | > 90
+
+|==============================================
+
+[discrete]
+=== Preview risky entities
+
+In {stack} version 8.11 and newer, you can preview risky entities before installing the latest risk engine. The preview shows the riskiest hosts and users found in the 1000 sampled entities during the time frame selected in the date picker.
```

Comment:
Again, I don't think we need to specify version # here, because this will only appear in the 8.11 and later docs.
```suggestion
You can preview risky entities before installing the risk engine. The preview shows the riskiest hosts and users found in the 1000 sampled entities during the time frame selected in the date picker.
```